plugins/org.jkiss.dbeaver.ext.bigquery/OSGI-INF/l10n/bundle_zh.properties

```diff
@@ -0,0 +1,5 @@
+# Translated By 2023 thenext600 (1556746401@QQ.com)
+
+Bundle-Vendor = DBeaver Corp
```

Comment:
The same is here. There is no point to duplicate it. The value will be taken from the default property file with English translation. Please remove such entries from everywhere.

Reply:
Copy that. Working on the problem you mentioned